listener/upload.go

```diff
@@ -205,6 +212,7 @@ func updateSystemPlatform(tx *gorm.DB, inventoryID string, accountID int, host *
 		CulledTimestamp:       host.CulledTimestamp.Time(),
 		Stale:                 staleWarning != nil && staleWarning.Before(time.Now()),
 		ReporterID:            getReporterID(host.Reporter),
+		YumUpdates:            yumUpdates,
```

Comment:
I am not 100% sure if we are null safe here but tests passed and code is cleaner now.